### PR TITLE
Added image caching

### LINKED ISSUE #39: Proper caching
1. Originals are not cached at all
2. Files on AWS are cached for a month _from the time of uploading_

Both should send an etag with an sha value, since the data is immutable and may be cached forever

### PR DESCRIPTION
**Pull request by rogierslag**
Closes #39 

@wouter-willems this caches :all_the_things:

I still have to test it thoroughly but will notify you when this is in the Docker containers

### CODE REVIEW

**Comment by wouter-willems** [comment]
sweet, :+1:

**Comment by rogierslag** [comment]
@wouter-willems can you recheck after some modifications?

File: dist/resize.js [hunk 1]
```diff
@@ -296,6 +310,9 @@ function startServer() {
   app.use(helpers.allowCrossDomain);
   app.get('/healthcheck', helpers.serverStatus);
   app.get('/robots.txt', helpers.robotsTxt);
+  app.get('/favicon.ico', function (req, res) {
+    helpers.send404(res, 'favicon.ico');
```
Comment (wouter-willems) [inline dist/resize.js]: Why this?

**Comment by wouter-willems** [comment]
Answer the question then :+1:

File: dist/resize.js (hunk 1 above)
Comment (rogierslag) [inline dist/resize.js]: I got annoyed by the error indicating `favicon.ico` was not a valid request

**Pull request merged**